generated/csv/fields.csv

```diff
@@ -8,7 +8,11 @@ agent.id,keyword,core,8a4f500d,1.1.0-dev
 agent.name,keyword,core,foo,1.1.0-dev
 agent.type,keyword,core,filebeat,1.1.0-dev
 agent.version,keyword,core,6.0.0-rc2,1.1.0-dev
+as.number,long,extended,15169,1.1.0-dev
```

Comment:
@webmat Seems like `reusable.top_level: false` is not being honored. I assume that if it's `false` that this field would not exist?

Reply:
Created https://github.com/elastic/ecs/issues/495.